tests/cypress/integration/autocomplete.spec.js

```diff
@@ -155,7 +155,7 @@ describe('Autocomplete', () => {
 
 				cy.get(`${config.selectors.autocomplete.facet} a`).then((facetOptions) => {
 					const firstOption = facetOptions[0];
-					const optionURL = firstOption.href;
+					const optionURL = firstOption.href.replace(config.url,'');
```

Comment:
Is this change necessary?

Seems that the href may not always contain the `config.url`...

Reply:
Adding a <form> wrapper around the input element on mockup index.html file may help remove the need for this.